### PR TITLE
[DB/Quest] Gaining Acceptance

### PR DESCRIPTION
**Pull request by xDevICCI**
- QUEST: Gaining Even More Acceptance

##### CHANGES PROPOSED:

-  make quest repeteable !
-  add 25 reputation in database

###### ISSUES ADDRESSED:

Closes 

##### TESTS PERFORMED:

- in my server , one player reported this  !

https://www.youtube.com/watch?v=xn_MmJC5My4 video guide !

##### HOW TO TEST THE CHANGES:

- Go to NPC : Master Smith Burninate <The Thorium Brotherhood>
- Take Quest and complete material and see not repeteable

- In column no exist value 25 reputation

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1556311512999196000.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1556311512999196000');
+
+UPDATE `quest_template` SET `RewardFactionOverride1`=2500 WHERE `ID`=13662;
+
+UPDATE `quest_template` SET `flags`=1 WHERE `ID`=13662;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1556311512999196000.sql]: those two `UPDATE` statements can be merged into one:
```sql
UPDATE `quest_template` SET `RewardFactionOverride1`=2500, `flags`=1 WHERE `ID`=13662;
```

Comment (poszer) [reply data/sql/updates/pending_db_world/rev_1556311512999196000.sql]: Done, thanks

**Comment by comix1988** [comment]
I checked the two quests reported (both that of the title of the pr and that of the video) and both work correctly on the master branch, commit 15ffcc65aa1e5f59df3839203f190e55f78b8bd2.
I can't reproduce the video bug.

**Comment by poszer** [comment]
> I checked the two quests reported (both that of the title of the pr and that of the video) and both work correctly on the master branch, commit [15ffcc6](https://github.com/azerothcore/azerothcore-wotlk/commit/15ffcc65aa1e5f59df3839203f190e55f78b8bd2).
> I can't reproduce the bug in the video.

The problem isn't in quest. It can be done normally. Problem is with repeat , and reputation reward. Check again please

**Comment by comix1988** [comment]
wait....
- [Gaining Even More Acceptance ](https://wowwiki.fandom.com/wiki/Quest:Gaining_Even_More_Acceptance)(100 dark iron residue) is repeatable and assigns 625 reputation;
- [Gaining Acceptance](https://wowwiki.fandom.com/wiki/Quest:Gaining_Acceptance) (4 dark iron residue) is not repeatable and assigns 25 reputation; (wowwiki is wrong, the quest is not repeatable, look on wowhead or wowgaming)

Everything works properly.

**Comment by poszer** [comment]
> wait....
> 
> * [Gaining Even More Acceptance ](https://wowwiki.fandom.com/wiki/Quest:Gaining_Even_More_Acceptance)(100 dark iron residue) is repeatable and assigns 625 reputation;
> * [Gaining Acceptance](https://wowwiki.fandom.com/wiki/Quest:Gaining_Acceptance) (4 dark iron residue) is not repeatable and assigns 25 reputation; (wowwiki is wrong, the quest is not repeatable, look on wowhead or wowgaming)
> 
> Everything works properly.

It's not. First, reputation rewards is wrong, It's giving 27 instead of 25. Quest is repeatable on blizz. If you have account there, check.

**Comment by comix1988** [comment]
sorry but I have no blizz account. @poszer 
Now check the quest again. Sorry but the title confused me.

**Comment by comix1988** [comment]
I find no differences after applying the sql file. @xDevICCI 
in both cases the quest is not repeatable and assigns 25 reputation:
before


after

**Comment by xDevICCI** [comment]
In the database is not asigned the 25 reputation

**Comment by poszer** [comment]
Thanks @xDevICCI , @comix1988 and me :rofl:

**Pull request merged**